src/main/java/hu/oe/nik/szfmv/automatedcar/visualization/CourseDisplay.java

```diff
@@ -54,8 +55,8 @@ private void paintComponent(Graphics g, World world) {
      * @param world {@link World} object that describes the virtual world
      * @return the ready to render doubleBufferedScreen
      */
-    private BufferedImage renderDoubleBufferedScreen(World world) {
-        BufferedImage doubleBufferedScreen = new BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB);
+    private BufferedImage renderDoubleBufferedScreen(WorldManager world) {
+        BufferedImage doubleBufferedScreen = new BufferedImage(this.world.getWidth(), this.world.getHeight(), BufferedImage.TYPE_INT_ARGB);
```

Comment:
we have to inner variables named "width" and "height". i'm ok with not using them but be consistent and only use one of them. e.g.: in line 61 it is used as "width" and "height"